### ISSUE TITLE
Should we move to a pull request based workflow?

### ISSUE THREAD

**Issue by roosmaa** [RiseVision/rise-react-wallet#99]
Over the past few weeks there have been couple of times times that I would've liked to discuss some of the changes before they land in `master` (both mine and Tobias's).

I'm proposing that we should start a discussion around this topic and come up with a pull request based workflow that gives us a chance of reviewing changes and which would work for us. Not sure "every change needs to be absolutely reviewed" is the right fit for us either. If we implemented that it would mean during my off-week Tobias would build up a backlog of PRs and get eventually blocked by the lack of merging.

**Comment by roosmaa** [comment]
I guess the answer is "no", considering the lack of responses.

**Comment by vekexasia** [comment]
Actually i'd reopen this. I'd vote for "yes"

**Comment by vekexasia** [comment]
Just a quick note. We could set this repo up so that It REQUIRES pull requests to be approved before being merged into master which i think should be the way to go.
